porcupine/plugins/pluginmanager.py

```diff
@@ -79,43 +87,42 @@ def __init__(self, master: tkinter.Misc) -> None:
         self.treeview.pack(side="left", fill="both", expand=True)
         scrollbar.pack(side="right", fill="y")
 
-        for index, width in enumerate([100, 150, 180]):
+        for index, width in enumerate(_cols_width):
             self.treeview.column(index, width=width, minwidth=width)
 
-        self.treeview.heading(0, text="Name")
-        self.treeview.heading(1, text="Type")
-        self.treeview.heading(2, text="Status")
+        for index, header_title in enumerate(["Name", "Type", "Status"]):  # be consistent
```

Comment:
Comment doesn't make sense to me, but this is simple code, and doesn't require a comment IMO:
```suggestion
        for index, header_title in enumerate(["Name", "Type", "Status"]):
```

Reply:
lol, that wasn't intentional